Makefile

```
@@ -60,4 +61,4 @@ pull_translations:
 	  && atlas pull \
 	           translations/paragon/src/i18n/messages:paragon \
 	           translations/frontend-app-admin-portal/src/i18n/messages:frontend-app-admin-portal
-	$(intl_imports) paragon frontend-app-admin-portal
+	$(intl_imports) paragon
```

`frontend-app-admin-portal` is still needed for the `inlt-imports.js` command.

```suggestion
	$(intl_imports) paragon frontend-app-admin-portal
```